tools/hidl/hidlcomp.cpp

```diff
@@ -5925,6 +5925,15 @@ void EspServInfo::write_esp_binding()
         outf("\tif (context.getClientVersion()<=0)\n");
         outf("\t\tcontext.setClientVersion(%s);\n\n", defVer.str());
     }
```

Comment:
Picky, but simpler to do:

    bool hasDefVer = getMetaVerInfo(tags,"default_client_version",defVer);
    if (!hasDefVer)
        hasDefVer = getMetaVerInfo(tags,"version",defVer);
    if (hasDefVer)
    {
        outf("\tif (context.getClientVersion()<=0)\n");
        outf("\t\tcontext.setClientVersion(%s);\n\n", defVer.str());
    }